.github/workflows/deploy-to-gcp.yaml

```diff
@@ -1,16 +1,20 @@
 name: Deploy Xplorers API to GCP
-
-on: push
+on: 
+  push:
+  pull_request:
 
 permissions:
-  contents: read
   id-token: write
+  contents: write
```

Comment:
Is this to allow Github Actions to create a release?
<img width="1350" alt="image" src="https://github.com/user-attachments/assets/2d1150b8-9dc5-4aa7-9f24-a17e80f4ed81">

Reply:
Removed it Was thinking of a pattern to store Plan but baby steps first